platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx

```diff
@@ -12,12 +12,14 @@ const ThumbnailNoImage = ({
   modalityTooltip,
   onClick,
   onDoubleClick,
+  canReject,
+  onReject,
```

Comment:
Better to have a slot + children here instead of explicit props? Or a separate component?

Only things that jump out to me is that `canReject` and `onReject` feel specific to DCM4CHEE terminology. Is "rejection" an industry term? Does Remove/Delete make more sense here?

Definitely not a blocker. Just musing out loud.